From ac4d8b4d8beaddfa8dc38a6d7a30f78ed86e317b Mon Sep 17 00:00:00 2001
From: Dongwon Kim <dongwon.kim@intel.com>
Date: Thu, 1 Dec 2022 14:11:22 -0800
Subject: [PATCH 13/34] ui/egl-helpers: display EGL error if eglCreateImageKHR
 fails

Let QEMU display error code if fails to create EGL image out of the
guest scanout.

Signed-off-by: Dongwon Kim <dongwon.kim@intel.com>
---
 ui/egl-helpers.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/egl-helpers.c b/ui/egl-helpers.c
index 3a88245b67..d546252c5f 100644
--- a/ui/egl-helpers.c
+++ b/ui/egl-helpers.c
@@ -281,7 +281,7 @@ void egl_dmabuf_import_texture(QemuDmaBuf *dmabuf)
                               EGL_LINUX_DMA_BUF_EXT,
                               NULL, attrs);
     if (image == EGL_NO_IMAGE_KHR) {
-        error_report("eglCreateImageKHR failed");
+        error_report("eglCreateImageKHR failed: %d", eglGetError());
         return;
     }
 
-- 
2.30.2

